substratevm/src/com.oracle.graal.pointsto.standalone/src/com/oracle/graal/pointsto/standalone/StandalonePointsToAnalysis.java

```diff
@@ -70,7 +70,7 @@ public void initializeMetaData(AnalysisType type) {
     public void onTypeInitialized(AnalysisType type) {
         AnalysisMethod clinitMethod = type.getClassInitializer();
         if (clinitMethod != null && !addedClinits.contains(clinitMethod)) {
-            addRootMethod(clinitMethod, true).registerAsImplementationInvoked(type);
+            addRootMethod(clinitMethod, true).registerAsImplementationInvoked("static root method");
```

Comment:
@cstancu Your change here fails the standalone analysis tests, because in `PointsToAnalysisMethod#registerAsImplementationInvoked` it requires the reason must be `InvokeTypeFlow` or `String`.
So I change the reason.